doc/doc-requirements.txt

```diff
@@ -15,7 +15,7 @@ numpydoc
 sphinx!=4.4
 sphinx-gallery>=0.10.0
 tomli>=2.0.1
-pydata-sphinx-theme==0.13.3
+grg-sphinx-theme>=0.3
```

Comment:
Why was this changed? If this was required, should have been done on a separate PR: this is unrelated to the `allow_break` issue. Also, this was part of another commit: https://github.com/dipy/dipy/commit/c2c42c44569ec12a4cdf5ed8a97c45bb114d6b11

Reply:
Agree. It might have been from squashing some unrelated commit

Reply:
Yea, sorry about that. I updated the branch from upstream while working on the issue and then squashed it, because of that this also went into the PR.